src/analysis/souffle/authorization_logic.dl

```diff
@@ -26,19 +26,23 @@
 // A Principal that makes assertions in authorization logic
 .type Principal <: symbol
 
+// These relations need to be populated by the emitted authorization logic 
+// code. Because the emitted authorization logic code appends "says_" to the 
+// beginning of relations, these relations need to break the style 
+// conventions here.
+//
 // Some party asserts the claim that the label of an access path should be
 // downgraded by removing the referenced tag from the set.
-.decl saysDowngrades(speaker: Principal, p: AccessPath, t: Tag)
+.decl says_downgrades(speaker: Principal, p: AccessPath, t: Tag)
```

Comment:
Instead of messing with `taint.dl`, I was just thinking of just adding some rules like  `saysDowngrades(...) :- says_downgrades(...)`. We could add this in datalog_fact.h here: https://github.com/google-research/raksha/pull/161/files#diff-ac24c09d794c9c04efca399024ec64dfc514c4a37cac98e2460aaf5fd1b99d49R60

Reply:
Yeah, that's probably better.